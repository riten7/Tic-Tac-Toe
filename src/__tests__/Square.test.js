import React from "react";
import { mount } from "enzyme";
import Square from "../Square";

const props = {
  value: 'X',
  winnerClass: "",
}

describe("Component renders properly", () => {
  it("Test Square", () => {
    const wrapper = mount(<Square {...props} />);
    expect(wrapper.html()).toMatchSnapshot();
  });
});