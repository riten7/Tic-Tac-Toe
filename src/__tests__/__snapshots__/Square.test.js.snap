// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders properly Test Square 1`] = `"<button class=\\" square\\">X</button>"`;
